src/amr_criteria/refinement_package.hpp

```diff
@@ -37,13 +37,18 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin);
 template <typename T>
 TaskStatus Tag(T *rc);
 
-AmrTag CheckAllRefinement(MeshBlockData<Real> *rc);
-
-AmrTag FirstDerivative(const AMRBounds &bnds, const ParArray3D<Real> &q,
-                       const Real refine_criteria, const Real derefine_criteria);
-
-AmrTag SecondDerivative(const AMRBounds &bnds, const ParArray3D<Real> &q,
-                        const Real refine_criteria, const Real derefine_criteria);
```

Comment:
The corresponding code in the .cpp file can also go, doesn't it?

Reply:
You're right I must have missed this